common/app/views/support/HtmlCleaner.scala

```diff
@@ -429,7 +429,11 @@ case class TagLinker(article: Article)(implicit val edition: Edition, implicit v
         val unlinkedParas = paragraphs.filterNot(_.html.contains("<a"))
 
         // pre-filter paragraphs so we do not do multiple regexes on every single paragraph in every single article
-        val candidateParagraphs = unlinkedParas.filter(_.html.contains(keyword.name))
+        val containsKeywordParas = unlinkedParas.filter(_.html.contains(keyword.name))
+
+        // don't link tags contained in article's pullquotes
+        val isPullquote = (e: Element) => (e.tagName() == "aside" && e.hasClass("element-pullquote"))
+        val candidateParagraphs = containsKeywordParas.filterNot(_.parents.exists(ancestor => isPullquote(ancestor)))
```

Comment:
I am not sure about the performance implications here. What do you think @jennysivapalan?